.github/workflows/daily-smoke-test.yml

```diff
@@ -28,4 +28,45 @@ jobs:
   create-vsix-artifact:
     name: 'Upload VSIX as artifact'
     uses: ./.github/workflows/create-vsix-artifact.yml
+  # Step 4: Report any problems
+  report-problems:
+    name: 'Report problems'
+    runs-on: ubuntu-latest
+    needs: [build-scanner-tarball, smoke-test, create-vsix-artifact]
+    if: ${{ failure() || cancelled() }}
+    steps:
+      - name: Report problems
+        shell: bash
+        env:
+          IS_CRITICAL: ${{ contains(join(steps.*.outcome), 'failure') || contains(join(steps.*.outcome), 'skipped') }}
+          RUN_LINK: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
+        run: |
+          if [[ ${{ env.IS_CRITICAL }} == true ]]; then
```

Comment:
@stephen-carter-at-sf @jfeingold35 : This logic is shared across multiple actions. Once the other PR is also approved, I could extract into a shared yml.